Resources/public/js/views/ez-dashboardview.js

```diff
@@ -28,7 +33,35 @@ YUI.add('ez-dashboardview', function (Y) {
          */
         render: function () {
             this.get('container').setHTML(this.template());
+            this._attachedViewEvents.push(Y.on("windowresize", Y.bind(this._uiSetHeight, this)));
+
             return this;
+        },
+
+        /**
+         * Sets the source of the iframe to the value of the iframeSource attribute.
+         *
+         * @method _setIFrameSource
+         * @private
+         */
+        _setIFrameSource: function () {
+            this.get('container').one('.ez-dashboard-content').set('src', this.get('iframeSource'));
         }
+
+    }, {
+        ATTRS: {
+            /**
+             * Stores the iframe Source
+             *
+             * @attribute iframeSource
+             * @type String
+             * @default 'http://ez.no/in-product/eZ-Platform'
+             * @readOnly
+             */
+            iframeSource: {
+                value: 'http://ez.no/in-product/eZ-Platform',
```

Comment:
Could be the protocol used by platformui itself. That's easy to do in JS is it not ?

But this can nonetheless be fixed later. I'd like to merge if no -1.